README.md

```diff
@@ -8,10 +8,14 @@ Compatibility
 The following Grove compatible devices are supported in Python:
 
 * Dexter Industries:
-  * Distance Sensor
-  * Inertial Measurement Unit Sensor
-  * Light Color Sensor
-  * Temperature Humidity Pressure Sensor
+  * **[Distance Sensor](https://www.dexterindustries.com/shop/distance-sensor/)** - Distance Sensor for the GoPiGo, GrovePi, and BrickPi.  The distance sensor can be mounted to the GoPiGo Raspberry Pi robot with or without the servo package to enable rotation.  The sensor detects distances from obstacles and objects, giving your robot the ability to navigate.
+
+  * **[Inertial Measurement Unit Sensor](https://www.dexterindustries.com/shop/imu-sensor/)** - The IMU Sensor attaches to the GrovePi, GoPiGo and BrickPi to detect motion, orientation, and position of your robot. It has a compass, accelerometer, and gyroscope and allows you to build a BalanceBot.
```

Comment:
Might want to mention it's only the GoPiGo3 as it's not supported on the GoPiGo2